### PR TITLE
changes to migration scripts for adding feature flags during migrations

### PR DESCRIPTION
**Pull request by RajeevHejib**
These changes cover adding existing feature flags during migration https://www.pivotaltracker.com/story/show/98374700

### CODE REVIEW

File: db/migrate/20140507165854_create_features.rb [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
       t.boolean :active, default: false
     end
+
+    Feature.create(name: :volunteer_ops_create)
+    Feature.create(name: :volunteer_ops_list)
+
   end
+
+  def down
+    # Feature.find_by(name: :volunteer_ops_create).destroy
```
Comment (tansaku) [inline db/migrate/20140507165854_create_features.rb]: let's remove this commented out code

**Comment by tansaku** [comment]
great work @RajeevHejib - this is the right branch to pull request to, and note how I've edited the title and your description slightly.  The Pivotal tracker id makes sense in the commit message, but not elsewhere.

See my inline comment about commenting, but this could be good to go very soon.

**Comment by RajeevHejib** [comment]
Thanks @tansaku. That makes sense.  I will keep it in mind for future. I would like to remove the commented line.

**Comment by johnnymo87** [comment]
:+1:

**Pull request merged**